test/controllers/games_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class GamesControllerTest < ActionController::TestCase
+  test "should get show" do
+    get :show
+    assert_response :success
```

Comment:
Yeah, noticed that as well. :( 
Okay, sounds good. I will defer this till tomorrow then.